packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -1030,6 +1030,36 @@ class MemoryController extends DisposableController
     return serviceManager?.vm?.operatingSystem == 'android';
   }
 
+  /// All library dependencies for the app.
+  List<LibraryRef> get activeLibraries =>
+      serviceManager.isolateManager.selectedIsolateLibraries;
+
+  /// Source file name as returned from allocation's stacktrace.
+  /// Map source URI
+  ///    packages/flutter/src/widgets/image.dart
+  /// would map to
+  ///    package:flutter/lib/src/widgets/image.dart
+  // TODO(terry): Review with Ben pathing doesn't quite work the
+  //              source file has the lib/ maybe a LibraryRef could
+  //              be returned if in a library otherwise file:/// as today?
```

Comment:
you mean in a package not in a library.

Reply:
Packages are libraryRefs in the vm service packages are exposed in selectedIsolateLibraries. But I'll change the name in the comment to say packages.

Reply:
```
  // TODO(terry): Review with Ben pathing doesn't quite work the source
  //              file has the lib/ maybe a LibraryRef could be returned
  //              if in a package today all packages are file:///?
```

Reply:
Done.